OpenRA.Mods.Common/Traits/LinkClientManager.cs

```diff
@@ -259,7 +259,7 @@ public bool CanLinkTo(Actor target, bool forceEnter = false, bool ignoreOccupanc
 		public TraitPair<ILinkHost>? ClosestLinkHost(ILinkHost ignore, BitSet<LinkType> type = default,
 			bool forceEnter = false, bool ignoreOccupancy = false)
 		{
-			var clients = type.IsEmpty ? linkClients : AvailableLinkClients(type);
+			var clients = type.IsEmpty ? linkClients : AvailableLinkClients(type, forceEnter);
```

Comment:
Should this commit ("Fix forceEnter being ignored when looking for ClosestDock") be squashed into "Add RequireForceMoveCondition to DockClientManager"?